t3/common.py

```diff
@@ -47,3 +50,34 @@ def delete_root_rmg_log(project_directory: str) -> None:
     rmg_log_path = os.path.join(project_directory, 'RMG.log')
     if os.path.isfile(rmg_log_path):
         os.remove(rmg_log_path)
+
+
+def get_species_by_label(label: str,
+                         species_list: list,
+                         ) -> Union[Species, None]:
+    """
+    Get a species from a list of species by its label. This label includes the index in parenthesis as assigned
+    in the chemkin file. Ex: H(3)
+
+    Args:
+        label (str): A species' label (species.label).
+        species_list (list): Entries are RMG Species objects.
+
+    Raises:
+        InputError: If ``label`` is None.
+
+    Returns:
+        Union[Species, None]: The corresponding species from the species_list.
+                              Returns ``None`` if no species was found.
+    """
+    if label is None:
+        raise InputError('Got None as label input')
```

Comment:
Do you mind raising a general error, e.g., `ValueError`? We shouldn't use errors imported from RMG for non-RMG stuff (we have several instances like that on master, let's gradually change them when we catch them)

Reply:
Updated